### PR TITLE
Add the Intel NUC13 to the recommended hardware list; deprecate NUC10

### PR DESCRIPTION
**Pull request by nathandyer**
## Status

Ready for review (although @zenmonkeykstop may prefer to hold off on merging until he is able to test the system as well)

## Description of Changes

* Description: This PR adds the Intel NUC13 to the recommended hardware list, including the hardware end-of-life table, and provides an updated photo showing the wireless card removal.

* Related Issues
https://github.com/freedomofpress/securedrop-docs/issues/480

## Testing
- [ ] Visual review
- [ ] CI passes

## Release 
* Can be merged and tagged in a stable release at any point (does not need to coincide with a server release)

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:454]: Should we call this the "ASUS 13th-gen NUC" since that's what the product page says? And then I suppose a small comment explaining that Intel transitioned NUCs to ASUS?

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:452]: Note that there was a typo in this anchor but I think it's fine to fix

**Review by legoktm — CHANGES_REQUESTED** [review]

File: docs/admin/installation/hardware.rst
Comment (legoktm) [reply docs/admin/installation/hardware.rst:454]: Hmmm, you removed "Intel" from everywhere in https://github.com/freedomofpress/securedrop-docs/pull/540 so seems like the answer is yes.

**Comment by legoktm** [comment]
I'll pick this up after #540 lands to avoid conflicting.

File: docs/admin/installation/hardware.rst
Comment (nathandyer) [reply docs/admin/installation/hardware.rst:452]: Thanks! Looks like it's fixed in the latest version of the branch.

File: docs/admin/installation/hardware.rst
Comment (nathandyer) [reply docs/admin/installation/hardware.rst]: Good call @legoktm! It looks like https://github.com/freedomofpress/securedrop-docs/pull/542/commits/8c5d7017f0e37539d647958121ddd8834ab6027c took care of the name switch, and then I just tacked on https://github.com/freedomofpress/securedrop-docs/pull/542/commits/55568e2286ec486c79e432d729f536e101388dfa which should cover off the explanation about the transition to ASUS. Please let me know what you think.

**Review by zenmonkeykstop — APPROVED** [review]
Fixed conflicts, approving pending CI run.

**Review by legoktm — APPROVED** [review]

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:477]: Why was this anchor removed? It was intentionally added in 5f098d78fb98ac8c9c3a021aad0f5730fe52204e.

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:453]: Why was this anchor removed? It was intentionally added in 5f098d78fb98ac8c9c3a021aad0f5730fe52204e.

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:431]: 10-12th is a bit awkward, I think "10th through 12th generation..." would read better?

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:455]: Can we avoid calling it "ASUS ..." after we removed "Intel" from the other headers?

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:506]: The intel link is a 404, while the removed ASUS link is fine.

File: docs/admin/installation/hardware.rst
Comment (legoktm) [inline docs/admin/installation/hardware.rst:735]: This part is wrong too

**Review by legoktm — CHANGES_REQUESTED** [review]
I think something went wrong in the rebase, a number of changes in the previous PR are being overwritten here.

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst:477]: I've standardized on the "nuc[x]_recommendation" format. This will break some historical anchors, but given the amount of time that's passed that's IMO fine.

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst:453]: (As above)

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst]: I reworded that entire intro paragraph a bit.

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst]: Done.

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst]: Switched back to ASUS link.

File: docs/admin/installation/hardware.rst
Comment (eloquence) [reply docs/admin/installation/hardware.rst]: Updated to ASUS.

**Comment by eloquence** [comment]
This should be ready for another review pass. Thanks Nathan for all the work on this.

**Review by legoktm — APPROVED** [review]
Thanks, LGTM

**Pull request merged**
